.github/workflows/test-tidy.yml

```diff
@@ -32,9 +32,11 @@ jobs:
 
       - name: Run Bazel tidy
         shell: bash
-        run: |
-          bazelisk run //bazel/ci:tidy
-          bazelisk run //:gazelle-check
+        run: bazelisk run //:tidy
+
+      - name: Run Bazel check
+        shell: bash
+        run: bazelisk run //:check
 
       - name: Check if tidy made modifications
         id: tidycheck
```

Comment:
Cannot comment on the unchanged code below..

nit for consistency on line 55:
```diff
- echo "::error::The repo is not tidy. Please run 'bazel run bazel/ci:tidy' and commit the changes."
+ echo "::error::The repo is not tidy. Please run 'bazel run //:tidy' and commit the changes."
```

Reply:
I've also moved the check step after this one.